src/server/controllers/jokeController.js

```diff
@@ -2,33 +2,51 @@ const sql = require('../../db/db');
 
 const jokeController = {};
 
+// get a joke from the database that the requesting user did not write
 jokeController.getJoke = async (req, res, next) => {
   try {
-    const response = await sql`SELECT * FROM jokes`;
-    console.log('joke', response[0].content);
-    res.locals.joke = response[0].content;
+    // temporary hard-coded user
+    const user = 'paloma'
+    // use once front-end sends user
+    // const { user } = req.body.user;
+    // retrieve the user's id
+    const userIdResult = await sql`SELECT id FROM users WHERE username=${user}`;
+    // handle id retrieval error
+    if (userIdResult.length === 0) return next({ log: `No user named ${user} found`, message: 'An error occured looking up this user'});
+    const userId = userIdResult[0].id;
+    // retrieve joke
+    const jokeResponse = await sql`SELECT content FROM jokes WHERE creator_id != ${userId} ORDER BY RANDOM() LIMIT 1`;
```

Comment:
I think we'll need the id of the joke in addition to the content of the joke to send back to the client so that we know which jokes the client is liking or disliking.